modules/options-for-notebook-server-environments.adoc

```diff
@@ -157,13 +157,31 @@ a| * Python 3.9
 * Scikit-learn 1.2
 * SciPy 1.10
 
+.2+| HabanaAI
```

Comment:
Not sure about the `.2+` here - TrustyAI doesn't have it and it's also just one version image :shrug:

Reply:
It's an AsciiDoc formatting thing.

Reply:
Will try and remove it.

Reply:
Removed.